aliasing/api/context.py

```diff
@@ -120,6 +121,8 @@ def __init__(self, channel):
         self._name = str(channel)
         self._id = channel.id
         self._topic = channel.topic if not isinstance(channel, discord.DMChannel) else None
+        self._category = AliasCategory(channel.category) if hasattr(channel, 'category') \
+                                                            and channel.category is not None else None
```

Comment:
could be simplified to `getattr(channel, 'category', None)`

Reply:
TIL `getattr` has a default section

```py
self._category = AliasCategory(channel.category) if getattr(channel, 'category', None) is not None else None
```

Would be the proper way to apply this, correct?